app/services/provider_interface/application_data_export.rb

```diff
@@ -50,7 +50,7 @@ def self.call(application_choices:)
           'award_year' => application.first_degree.award_year,
           'institution_details' => application.first_degree.institution_name,
           'equivalency_details' => replace_smart_quotes(application.first_degree.composite_equivalency_details),
-          'awarding_body' => application.first_degree.awarding_body,
+          'awarding_body' => nil,
```

Comment:
Sorry, only just thought that this is potentially worth a comment explaining why this and the above look odd